tests/kickstarts/test_suite.cfg

```diff
@@ -68,31 +79,55 @@ clearpart --linux --initlabel
 # Create primary system partitions (required for installs)
 part /boot --fstype=xfs --size=512
 part pv.01 --grow --size=1
-volgroup VolGroup pv.01
-logvol / --fstype=xfs --name=LogVol01 --vgname=VolGroup --size=3072
-logvol /home --fstype=xfs --name=LogVol02 --vgname=VolGroup  --size=1024
-logvol /tmp --fstype=xfs --name=LogVol03 --vgname=VolGroup --size=1024
-logvol /var --fstype=xfs --name=LogVol04 --vgname=VolGroup --size=2048
-logvol /var/log --fstype=xfs --name=LogVol05 --vgname=VolGroup --size=512
-logvol /var/log/audit --fstype=xfs --name=LogVol06 --vgname=VolGroup --size=512
-logvol swap --name=lv_swap --vgname=VolGroup --size=1024
+
+# Create a Logical Volume Management (LVM) group (optional)
+volgroup VolGroup --pesize=4096 pv.01
+
+# Create particular logical volumes (optional)
+logvol / --fstype=xfs --name=root --vgname=VolGroup --size=12288 --grow
+# CCE-26557-9: Ensure /home Located On Separate Partition
+logvol /home --fstype=xfs --name=home --vgname=VolGroup --size=1024 --fsoptions="nodev"
+# CCE-26435-8: Ensure /tmp Located On Separate Partition
+logvol /tmp --fstype=xfs --name=tmp --vgname=VolGroup --size=1024 --fsoptions="nodev,nosuid,noexec"
+# CCE-26639-5: Ensure /var Located On Separate Partition
+logvol /var --fstype=xfs --name=var --vgname=VolGroup --size=2048 --fsoptions="nodev"
+# CCE-26215-4: Ensure /var/log Located On Separate Partition
+logvol /var/log --fstype=xfs --name=log --vgname=VolGroup --size=1024 --fsoptions="nodev,nosuid,noexec"
+# CCE-26436-6: Ensure /var/log/audit Located On Separate Partition
+logvol /var/log/audit --fstype=xfs --name=audit --vgname=VolGroup --size=512 --fsoptions="nodev,nosuid,noexec"
+logvol swap --name=swap --vgname=VolGroup --size=2016
 
 # Packages selection (%packages section is required)
 %packages
 
-# requirements of SSG Test Suite
+# Install selected additional packages (required by PCI-DSS profile)
+# CCE-27024-9: Install AIDE
+aide
+
+# Install libreswan package
+libreswan
+
 openscap-scanner
 tar
 qemu-guest-agent
+openssh-clients
+openssh-server
 
 %end # End of %packages section
 
-%post --log /root/qemu-guest.log
+%post --log /root/post-install.log --interpreter /bin/bash
 
 # initialize guest agent for SSG Test Suite
 systemctl enable qemu-guest-agent.service
 systemctl start qemu-guest-agent.service
 
+ssh-keygen -N '' -t rsa -f /etc/ssh/ssh_host_rsa_key
+mkdir -p /root/.ssh
+printf "%s\n" "&&HOST_PUBLIC_KEY&&" >> /root/.ssh/authorized_keys
+chmod og-rw /root/.ssh /root/.ssh/authorized_keys
+sed -i '/session\s\+required\s\+pam_loginuid.so/d' /etc/pam.d/sshd
```

Comment:
What is the reason to remove `pam_loginuid.so`?

Reply:
Actually, this whole part (including ssh key generation) is from our Dockerfiles and it is true that it should not be needed. I will remove it and test if everything works as expected. Thanks